Common/ac/spritecache.cpp

```diff
@@ -53,6 +53,28 @@ SpriteInfo::SpriteInfo()
 {
 }
 
+int SpriteInfo::GetAdjustedWidth(bool hires_ctx) const
+{
+    if (!IsVarRes())
+        return Width;
+    if (IsHiRes() && !hires_ctx)
```

Comment:
Probably don't need the elses if you're just returning

Reply:
I realized that these functions are wrong for some reason, and I am not using them anyway, so going to remove them.